src/cloud/components/Views/Table/TableViewContentManager.tsx

```diff
@@ -596,4 +628,8 @@ const Container = styled.div`
     margin: 0 !important;
     width: 100%;
   }
+
+  .rc-select .form__select__single-value {
```

Comment:
Can you switch the rc-select className to a more readable one? like `sorting-options__select` etc..